addons/docs/src/lib/docgen/flow/createType.ts

```diff
@@ -40,8 +40,7 @@ function generateUnion({ name, raw, elements }: DocgenFlowUnionType): PropType {
   }
 
   if (raw != null) {
-    // Flow Unions can be defined with or without a leading `|` character, so try to remove it.
-    return createSummaryValue(raw.replace(/^\|\s*/, ''));
```

Comment:
I don't think I changed this on purpose, might be a merge conflict that was incorrectly resolved.